suite-native/storage/src/tests/migrations/walletSettingsV1.test.ts

```diff
@@ -1,22 +1,198 @@
+import { getStoredState } from 'redux-persist';
+
 import { NetworkSymbol } from '@suite-common/wallet-config';
 
-import { migrateDiscoveryConfigToWalletSettings } from '../../migrations/walletSettings/v1';
+jest.mock('../../storage', () => ({
+    initMmkvStorage: jest.fn().mockResolvedValue({}),
+}));
+
+jest.mock('redux-persist', () => ({
+    getStoredState: jest.fn(),
+}));
+
+import { initialMigrateAppSettingsAndDiscoveryConfig } from '../../migrations/walletSettings/v1';
+
+const mockGetStoredState = getStoredState as jest.MockedFunction<typeof getStoredState>;
+
+describe(initialMigrateAppSettingsAndDiscoveryConfig.name, () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should migrate local currency from fiatCurrencyCode', async () => {
+        mockGetStoredState.mockImplementation(({ key }) => {
+            if (key === 'appSettings') {
+                return Promise.resolve({
+                    fiatCurrencyCode: 'czk',
+                });
+            }
+
+            return Promise.resolve(undefined);
+        });
+
+        const migratedState = await initialMigrateAppSettingsAndDiscoveryConfig(undefined);
+
+        expect(migratedState).toEqual({
+            localCurrency: 'czk',
+        });
+    });
+
+    it('should ignore non-existent currency from fiatCurrencyCode', async () => {
+        mockGetStoredState.mockImplementation(({ key }) => {
+            if (key === 'appSettings') {
+                return Promise.resolve({
+                    fiatCurrencyCode: 'mycoin',
+                });
+            }
+
+            return Promise.resolve(undefined);
+        });
+
+        const migratedState = await initialMigrateAppSettingsAndDiscoveryConfig(undefined);
+
+        expect(migratedState).toEqual({});
+    });
+
+    it('should migrate local currency from fiatCurrency.label', async () => {
+        mockGetStoredState.mockImplementation(({ key }) => {
+            if (key === 'appSettings') {
+                return Promise.resolve({
+                    fiatCurrency: { label: 'czk', value: 'Czech koruna' },
+                });
+            }
+
+            return Promise.resolve(undefined);
+        });
+
+        const migratedState = await initialMigrateAppSettingsAndDiscoveryConfig(undefined);
+
+        expect(migratedState).toEqual({
+            localCurrency: 'czk',
+        });
+    });
+
+    it('should migrate bitcoin amount satoshi units', async () => {
+        mockGetStoredState.mockImplementation(({ key }) => {
+            if (key === 'appSettings') {
+                return Promise.resolve({
+                    bitcoinUnits: 3, // SATOSHI
```

Comment:
There should be no problem to `import { PROTO } from '@trezor/connect';`, is there?
```suggestion
                    bitcoinUnits: PROTO.AmountUnit.SATOSHI,
```

Reply:
b7264f9d2aca80d2a93e69c7409c301a2cd78a16